src/components/capture/useDiscoverCapture.ts

```diff
@@ -326,7 +331,21 @@ function useDiscoverCapture(
                     options?.initiateRediscovery ||
                     options?.initiateDiscovery
                 ) {
-                    const draftsResponse = await createEntityDraft(entityName);
+                    // If we are doing an initial discovery add the name name to the name
+                    // If not we are either refreshing collections during create OR during edit
+                    //  Refreshing during:
+                    //    create requires draftedEntityName because it has the connector image added to it
+                    //    edit   requires entityName        because it is the name already in the system and
+                    //                                        we do not have a draftedEntityName yet
+                    const processedEntityName = options.initiateDiscovery
+                        ? `${entityName}/${stripPathing(imageName)}`
+                        : hasLength(draftedEntityName)
+                        ? draftedEntityName
+                        : entityName;
+
+                    const draftsResponse = await createEntityDraft(
+                        processedEntityName
+                    );
```

Comment:
This handles more cases than before. We need to make sure that initial discover, discovering new collections during create, and discovering new collections during edit all work and do not stack up the imageName.


* The comment looks much nicer in an IDE